### ISSUE TITLE
Add information on ggplot2 and dplyr Cheatsheet availability from the dropdown menu in RStudio?

### ISSUE THREAD

**Issue by anamtk** [datacarpentry/R-ecology-lesson#679]
I recently discovered that both the dplyr and ggplot2 cheatsheets are now a part of the RStudio dropdown menu ("Help" -> "Cheatsheets"). I used to have to do an internet search for these every time I used them! This could be a nice thing to point out in the lessons on dplyr and ggplot2 (or perhaps the "finding help" lessons?), especially for users who like the non-coding feel of dropdown menus offered by RStudio!

**Comment by Teebusch** [comment]
Thank you for this suggestion! The cheat sheets are being mentioned at the very beginning of the workshop, in `Before we start > Seeking help` [(link)](https://datacarpentry.org/R-ecology-lesson/00-before-we-start.html#Package_vignettes_and_cheat_sheets)

The text is:

> There is also a Help menu at the top of the RStudio window, that has cheat sheets for popular packages, RStudio keyboard shortcuts, and more.

It doesn't mention dplyr or ggplot2 explicitly but instructors could certainly refer back to the cheat sheets later in the workshop as they see fit. Do you think there should be additional reminders for the cheat sheets in the ggplot and dplyr lessons?

**Comment by tobyhodges** [comment]
Thanks both for contributing to this discussion. The lesson underwent a major update and reorganisation when https://github.com/datacarpentry/R-ecology-lesson/pull/887 was merged. As this issue relates to content in a version of the lesson before that update took place, I will close it. Please open a new issue if you believe that some or all of the changes being discussed here are remain relevant to the redesigned lesson, linking to this thread where appropriate.
